services/worker-manager/schemas/v1/workertype-list

```diff
@@ -0,0 +1,27 @@
+$schema: "/schemas/common/metaschema.json#"
+title: WorkerType Full
+description: A complete workerType definition
+type: object
+properties:
+  data:
+    type: array
+    title: "List of all workertypes"
+    description: List of all workertypes
+    uniqueItems: true
+    items: {$ref: 'workertype-full.json#'}
+
+  continuationToken:
+    type:               string
+    title:              "Continuation Token"
+    description: |
+      Opaque `continuationToken` to be given as query-string option to get the
+      next set of worker-types in the worker-manager.
+      This property is only present if another request is necessary to fetch all
+      results. In practice the next request with a `continuationToken` may not
+      return additional results, but it can. Thus, you can only be sure to have
+      all the results if you've called `listWorkerTypes` with `continuationToken`
+      until you get a result without a `continuationToken`.
+
+additionalProperties: true
```

Comment:
This should be false, right?

Reply:
https://github.com/taskcluster/taskcluster/pull/632#discussion_r279103307 
I understand there also will be `continuation` prop